static/src/javascripts/projects/commercial/modules/dfp/googletag-cmd.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import mediator from 'lib/mediator';
+
+const GOOGLETAG_CMD_STUBBED = 'googletag-cmd-stubbed';
+
+export const stubGoogletagCmd = (): void => {
+    window.googletag = {
+        cmd: [],
+    };
+    mediator.emit(GOOGLETAG_CMD_STUBBED);
+};
+
+export const awaitGoogletagCmd: Promise<void> = new Promise(resolve =>
+    mediator.once(GOOGLETAG_CMD_STUBBED, resolve)
+);
```

Comment:
Same as above (sorry)

I think following the pattern of initialising a module in the commercial bootstrap will solve the need to await. So you will probably no longer need this 😄